src/drivers/intel/cavs/hda-dma.c

```diff
@@ -218,6 +219,12 @@ static void hda_dma_get_dbg_vals(struct dma_chan_data *chan,
 #define hda_dma_ptr_trace(...)
 #endif
 
+static void hda_dma_l1_exit_notify(void *arg, enum notify_id type, void *data)
+{
+	/* Force Host DMA to exit L1 */
+	pm_runtime_put(PM_RUNTIME_HOST_DMA_L1, 0);
```

Comment:
Currently, the DMI L1 pm-runtime context is not ref-counted which may lead to a situation when shorter LL processing on one core puts L1 when longer one on another is still LL-processing (thing for another patch/PR).

Reply:
Multicore synchronization added by implementing simple ref counting.